source/03-specifications/03-counter-report-common-attributes-and-elements.rst

```diff
@@ -526,6 +526,11 @@ Access Types
 
 In order to track the value of usage for licensed content, librarians want to know how much Open Access or other freely available content was used and how much content was behind a paywall. To accommodate this R5 has introduced an Access_Type attribute with values of Controlled, OA_Gold, OA_Delayed, and Other_Free_To_Read. The table below lists the Access_Types and the Host_Types and reports they apply to. Note that Access_Type relates to access on the platform where the usage occurs: if access to a Gold Open Access article is restricted on a platform (for example because the article is included in an aggregated full-text database available to subscribers only) the Access_Type is Controlled.
 
+Note that the values for Access_Type changed in R5.1 to reflect community needs around reporting and to address common misunderstandings. 
+
+* Open was previously known as OA_Gold. 
```

Comment:
This seems misleading to me - OA_Gold is covered by Gold, but Gold includes much more than just OA_Gold.

The paragraph above the note still lists the old Access_Types. If we want to keep that, I would suggest to add the note about the changes after that sentence, and then start a new paragraph with "The table below...".